postfix/tasks/installation.yml

```diff
@@ -2,30 +2,27 @@
 
 - name: remove conflicting packages
   apt:
-    name: '{{ item }}'
+    name: '{{ postfix_packages_removed }}'
```

Comment:
Variable name should be aligned, either use `removed` or `remove`.

Reply:
https://github.com/adfinis-sygroup/ansible-roles/pull/23